src/applications/simple-forms/mock-simple-forms-patterns/arrayBuilder/helpers.js

```diff
@@ -52,19 +52,6 @@ export function onNavBackKeepUrlParams({ goPreviousPath, urlParams }) {
   goPreviousPath(urlParams);
 }
 
-/**
- * Gets the query params from the URL and parses them as an object
- *
- * @param {string} [queryString] optional e.g. `?add=true`
- * @returns {object} e.g. `{ add: 'true' }`
- */
-export function parseQueryParams(queryString = window.location?.search) {
```

Comment:
removed this because I realized there was this native helper instead:
`new URLSearchParams(window.location?.search);`